src/scanner/get-rule-inclusions.ts

```diff
@@ -24,6 +24,10 @@ export const explicitRuleOverrides: DictionaryStringTo<RuleIncluded> = {
         status: 'excluded',
         reason: 'Based on feedback from users we tested to check the user impact of duplicate ID failures on static elements. We found no user impact on the experience with any of the ATs including Narrator, JAWS, NVDA, Dragon, and Windows Speech Recognition. After further discussions with the Tooling Advisory Board, we decided to make this a best practice rule. See #4102.',
     },
+    'label-content-name-mismatch': {
```

Comment:
Per the big comment I left in filter-results.ts, I don't think this should actually be included at all here, but it's easy to see how you'd think it should be; this file doesn't make itself clear that it only applies to automated checks. It would be a good idea to improve the naming and/or leave a comment to make it more clear (and to fix the very misleading comments on `empty-table-header` and `form-field-multiple-labels`; they should probably stop at "only reports needs-review results", it's misleading to imply that we'd want to include them once we implement needs-review content for them).